dtbase/webapp/app/sensors/templates/readings.html

```diff
@@ -0,0 +1,146 @@
+                <thead>
+                  <tr>
+                    <th>Timestamp</th>
+		    {%
+                    for m in measure_names:
+                    %}
+                    <th>{{m}}</th>
+		    {% endfor %}
+                  </tr>
+                </thead>
+                <tbody>
+		  {% for sd in sensor_data: %}
+		  <tr>
+		    <th>{{ sd['timestamp'] }}</th>
+		    {% for m in measure_names: %}
+		    <th>{{ '%0.2f' | format(sd[m]|float) }} </th>
+		    {% endfor %}
+		  </tr>
+		  {% endfor %}
+                </tbody>
+              </table>
+	      <form method="post">
+		<input type="submit" value="Download" name="download"/>
+	      </form>
+            </div>
+          </div>
+        </div>
+      </div>
+      {% endif %}
+    </div>
+  </div>
+{% endblock content %}
+
+{% block javascripts %}
+  {{ super() }}
+  <!-- Datatables -->
+  <script src="{{ url_for('static', filename='node_modules/datatables.net/js/jquery.dataTables.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-bs/js/dataTables.bootstrap.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-buttons/js/dataTables.buttons.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-buttons-bs/js/buttons.bootstrap.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-buttons/js/buttons.flash.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-buttons/js/buttons.html5.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-buttons/js/buttons.print.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-fixedheader/js/dataTables.fixedHeader.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-keytable/js/dataTables.keyTable.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-responsive/js/dataTables.responsive.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-responsive-bs/js/responsive.bootstrap.js') }}"></script>
+  <script src="{{ url_for('static', filename='node_modules/datatables.net-scroller/js/dataTables.scroller.js') }}"></script>
+
+  <script>
+
+    $(document).ready(function() {
+        $('#datatable-responsive').DataTable( {
+            "order": [[ 0, "desc" ]]
+        } );
+    } );
+
+    $(window).load(function() {
+
+        $('#reportrange span').html("{{ dt_from }}" + ' - ' + "{{ dt_to }}");
+
+    });
+
+    $('#reportrange').on('apply.daterangepicker', function(ev, picker) {
+        var request_url = "/readings/aranet_trh?range=" + picker.startDate.format('YYYYMMDD') + "-" + picker.endDate.format('YYYYMMDD')
+
+        location.replace(request_url)
+    });
+
+</script>
```

Comment:
I'm wondering whether we should keep js in static/javascript, though this is quite short so probably fine in here.